test/connectors/VatSubscriptionConnectorSpec.scala

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package connectors
+
+import base.BaseSpec
+import mocks.MockHttp
+import assets.TestConstants.vrn
+import assets.CustomerDetailsTestAssets._
+import connectors.httpParsers.ResponseHttpParsers.HttpGetResult
+import models.{CustomerDetails, MandationStatus}
+import common.MandationStatuses.nonMTDfB
+import play.api.http.Status
+import uk.gov.hmrc.http.HttpResponse
+
+import scala.concurrent.Future
+
+class VatSubscriptionConnectorSpec extends BaseSpec with MockHttp {
+
+  val errorModel = HttpResponse(Status.BAD_REQUEST, responseString = Some("Error Message"))
+
+
+  object TestVatSubscriptionConnector extends VatSubscriptionConnector(mockHttp, mockAppConfig)
+
+  "VatSubscriptionConnector" when {
+
+    "calling the 'vatSubscriptionUrl' method" should {
+
+      "format the url correctly" in {
+
+        val testUrl = TestVatSubscriptionConnector.vatSubscriptionUrl(vrn, "endpoint")
+        testUrl shouldBe s"${mockAppConfig.baseUrl("vat-subscription")}/vat-subscription/999999999/endpoint"
+      }
+    }
+
+    "calling 'getCustomerDetails'" when {
+
+      def result: Future[HttpGetResult[CustomerDetails]] = TestVatSubscriptionConnector.getCustomerDetails(vrn)
+
+      "called for a Right with CustomerDetails" should {
```

Comment:
This wording is confusing as is the one on line 61 - could you reword it to be like the example on line 74 please?